opennmt/models/language_model.py

```diff
@@ -221,16 +229,21 @@ def make_training_dataset(self,
     dataset = self.make_dataset(features_file, training=True)
     if weights is not None:
       dataset = (dataset, weights)
+    transform_fns = []
+    map_func = lambda x: self.make_features(element=x, training=True)
+    transform_fns.append(lambda dataset:
+                           dataset.map(map_func))
+    transform_fns.append(dataset_util.filter_examples_by_length(
+                         maximum_features_length=maximum_features_length,
+                         maximum_labels_length=maximum_labels_length,
```

Comment:
`maximum_labels_length` does not need to be passed for unsupervised datasets.

Reply:
indeed, removed.

Reply:
actually, it is not that simple. removing the parameters make a style warning because `maximum_labels_length` is not used. And removing from the parameter list of `make_training_dataset`, make the generic call in runner.py fails with extraneous parameter error. I put it back as it was.